### ISSUE TITLE
Major problems - anyone else have disappearing content?

### ISSUE THREAD

**Issue by codymckibb** [ThemeFuse/Unyson#782]
So, everything I can find on the internet says that ThemeFuse are the developers of this plugin. However, when I contact them directly, they keep giving me the run-around. Unfortunately the Unyson plugin has deleted loads of content that I put in their shortcodes and seems nobody at ThemeFuse wants to help.

This is urgent, and it's affecting my 600+ members in my private membership site.

First off, I'm on WPEngine, and not sure if that or the latest WP update have anything to do with it …

But this plugin just DOES NOT work, nothing about it seems to agree with my setup. Biggest issue is I am running a big online course, and I went through and redesigned everything using Unyson shortcodes for buttons, downloads, notification boxes, etc. and this is the 2nd time they have ALL disappeared, all the shortcodes taking all their content with them (i.e., downloads, buttons, info boxes all disappearing, and I have no reference)

Additionally the map and other functions that my theme (Woffice) pulls from Unyson addons do not work properly.

The one thing I do not have much of is time, time to comb through more forums and support tickets and get faffed around by companies like ThemeFuse and to wait in the hopes that someone will help out of the goodness of their heart.  I am more than willing to pay for the right person's time to help me fix this ASAP.

Thanks

**Comment by moldcraft** [comment]
Do you know how to reproduce the problem, or it happens randomly?

**Comment by codymckibb** [comment]
No, I do not know how this happened or how to reproduce the bug.

I'm using the default WordPress editor, btw. And inserting a few of the
Unyson functions from the shortcodes dropdown.

For the purposes of this conversations, let's keep it simple by saying I'm
using an Alert Box or whatnot. You with me?

All I know is that on occasion, when I add an Alert Box, and put say a
paragraph of content in there, and hit the publish or update button, and
view the newly-updated page, about 2/10 times it does not retain the
paragraph of content I put in the box and shows up as an empty square.

I realized that right off the bat, so I was careful to double check
*always* to make sure that it had retained the content before moving on to
the next page. I went through my entire course and inserted these boxes
with Unyson shortcodes. And it was all there originally.

Now - all I can tell you is that at *some* point ALL that content got
erased. Everything that was in Alert Boxes, or buttons or etc. I had a
member tell me the buttons were broken, and I logged in to find that
*everything* enclosed within Unyson shortcodes is gone.

Now - I've noticed that when one switches to the HTML view in the WP
editor, there is no content in the shortcode, and it appears there is no
unique ID either, so I do not know how or where Unyson stores this
information. But - it seems like a poorly crafted solution to me, and I
hope to the gods of the north that I didn't just lose ALL that.

Thanks for any help.
